.github/workflows/no-response.yaml

```diff
@@ -0,0 +1,39 @@
+name: No Response
+
+# Both `issue_comment` and `scheduled` event types are required for this Action
+# to work properly.
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # Schedule for five minutes after the hour, every hour
+    - cron: '5 * * * *'
+
+# By specifying the access of one of the scopes, all of those that are not
+# specified are set to 'none'.
+permissions:
+  issues: write
+
+jobs:
+  noResponse:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: lee-dohm/no-response@9bb0a4b5e6a45046f00353d5de7d90fb8bd773bb
+        with:
+          token: ${{ github.token }}
+          # Comment to post when closing an Issue for lack of response. Set to `false` to disable
+          closeComment: >
```

Comment:
Do you mind using the text we worked out for the DartPad repo?

```
Without additional information we're not able to resolve this issue,
so it will be closed at this time. You're still free to add more info
and respond to any questions above, though. We'll reopen the case
if you do. Thanks for your contribution!
```

Reply:
Updated.